hazelcast/src/main/java/com/hazelcast/jet/impl/JetBootstrap.java

```diff
@@ -85,356 +36,15 @@
-        }
-
-        private Job remember(@Nonnull Job job) {
-            submittedJobs.add(job);
-            return job;
-        }
-
-        @Nonnull @Override
-        public LightJob newLightJobInt(Object jobDefinition) {
-            return instance.newLightJobInt(jobDefinition);
-        }
-
-        private JobConfig updateJobConfig(@Nonnull JobConfig config) {
-            if (jar != null) {
-                config.addJar(jar);
-            }
-            if (snapshotName != null) {
-                config.setInitialSnapshotName(snapshotName);
-            }
-            if (jobName != null) {
-                config.setName(jobName);
-            }
-            return config;
-        }
-
-        @Nonnull @Override
-        public List<Job> getJobs() {
-            return instance.getJobs();
-        }
-
-        @Override
-        public Job getJob(long jobId) {
-            return instance.getJob(jobId);
-        }
-
-        @Nonnull @Override
-        public List<Job> getJobs(@Nonnull String name) {
-            return instance.getJobs(name);
-        }
-
-        @Nonnull @Override
-        public <K, V> IMap<K, V> getMap(@Nonnull String name) {
-            return instance.getMap(name);
-        }
-
-        @Nonnull @Override
-        public <K, V> ReplicatedMap<K, V> getReplicatedMap(@Nonnull String name) {
-            return instance.getReplicatedMap(name);
-        }
-
-        @Nonnull @Override
-        public JetCacheManager getCacheManager() {
-            return instance.getCacheManager();
-        }
-
-        @Nonnull @Override
-        public <E> IList<E> getList(@Nonnull String name) {
-            return instance.getList(name);
-        }
-
-        @Nonnull @Override
-        public <T> ITopic<T> getReliableTopic(@Nonnull String name) {
-            return instance.getReliableTopic(name);
-        }
-
-        @Nonnull @Override
-        public <T> Observable<T> getObservable(@Nonnull String name) {
-            return instance.getObservable(name);
-        }
-
-        @Override
-        public void shutdown() {
-            instance.shutdown();
-        }
-
-        @Override
-        public boolean existsDistributedObject(@Nonnull String serviceName, @Nonnull String objectName) {
-            return instance.existsDistributedObject(serviceName, objectName);
-        }
-
-        @Override
-        public ILogger getLogger() {
-            return instance.getLogger();
-        }
-
-        @Override
-        public Job newJobProxy(long jobId) {
-            return instance.newJobProxy(jobId);
-        }
-
-        @Override
-        public Job newJobProxy(long jobId, Object jobDefinition, JobConfig config) {
-            return instance.newJobProxy(jobId, jobDefinition, config);
-        }
-
-        @Override
-        public List<Long> getJobIdsByName(String name) {
-            return instance.getJobIdsByName(name);
-        }
+    public static JetInstance getInstance() {
```

Comment:
This is in an `impl` package, we can remove it without breaking compatibility?

Reply:
I removed this class.